learn/installing-ballerina.md

```diff
@@ -72,20 +72,20 @@ Follow the steps below to obtain the Ballerina source code.
 
 1. Execute the below command to clone the ['ballerina-lang'](https://github.com/ballerina-platform/ballerina-lang) source repository.
 
-```
-git clone --recursive https://github.com/ballerina-platform/ballerina-lang.git
-```
->**Tip:** If you have already forked the repository to your GitHub account, then execute the below command replacing <YOUR-GITHUB-USERNAME> with your Git username.
+  ```
+  git clone --recursive https://github.com/ballerina-platform/ballerina-lang.git
+  ```
+  >**Tip:** If you have already forked the repository to your GitHub account, then execute the below command replacing <YOUR-GITHUB-USERNAME> with your Git username.
 
-```
-git clone --recursive https://github.com/<YOUR-GITHUB-USERNAME>/ballerina-lang.git
-```
+  ```
+  git clone --recursive https://github.com/<YOUR-GITHUB-USERNAME>/ballerina-lang.git
+  ```
```

Comment:
Can we add one more tab to the sections under each numbered item to make that part more clear?

Reply:
Done